core/templates/dev/head/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-or-update-solution-modal.template.html

```diff
@@ -1,5 +1,5 @@
 <div class="modal-header protractor-test-add-or-update-solution-modal">
-  <h3><[StateSolutionService.savedMemento !== null ? 'Update Solution' : 'Add Solution']></h3>
+  <h3><[StateSolutionService.savedMemento !== null ? 'Update Solution' : 'Add Solutions']></h3>
```

Comment:
Why did you change this? Each card has only one solution.

Reply:
Ah, sorry confused b/n the only and only yes they have the only solution, Thanks!